### PR TITLE
Docs add elasticsearch

### PR DESCRIPTION
**Pull request by niallcreech**

### CODE REVIEW

**Comment by puneetk** [comment]
@niallcreech :)

**Comment by niallcreech** [comment]
Sorry, having one of those days :)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 template formula
```
Comment (puneetk) [inline CHANGELOG.rst]: collectd formula ?

**Comment by genuss** [comment]
In this architecture it is not possible to turn on elasticsearch and redis simultaneously! And any other python plugin.

**Pull request merged**